src/main/kotlin/io/github/vootelerotov/renovator/Renovator.kt

```diff
@@ -96,7 +96,8 @@ class Renovator : CliktCommand() {
 
     if (
       checksClient.getCheckRuns(pr.head()?.ref()).get().checkRuns()
-        .any { it.conclusion().getOrNull() != CheckRunConclusion.success }
+        .mapNotNull { it.conclusion().getOrNull() }
```

Comment:
If you get back `null` from conclusion, we would filter it out and not fail on it. So, maybe `map` instead of `mapNotNull` so we would not try to proceed with merge if we get `null`  from conclusion?

Reply:
Ah indeed, was so busy looking at the `CheckRunConclusion` list that I missed that part. Good catch!